### PR TITLE
Add autoconnect priority to centos target node network config

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR solves issue of two connection profiles competing for eth0 interface. We set higher priority to eth0.nmconnection profile.
The dev-tools part  of the solution is [here](https://github.com/Nordix/metal3-dev-tools/pull/587)

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml [hunk 1]
```diff
@@ -104,6 +104,8 @@ files:
       interface-name=eth0
       master={{ IRONIC_ENDPOINT_BRIDGE }}
       slave-type=bridge
+      autoconnect=yes
+      autoconnect-priority=999
```
Comment (kashifest) [inline tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml:108]: 999 means highest priority? How did you come up with that number ?

Comment (Sunnatillo) [reply tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml:108]: not the highest number. It is int32. I used an example from [here](https://www.redhat.com/sysadmin/becoming-friends-networkmanager). Basically any number above 0 works in our case.

**Review by kashifest — APPROVED** [review]

**Pull request merged**